ming_sim/cli/terminal.py

```
@@ -551,70 +530,64 @@ def _trail_extraction_after_reply_cli(
+            try:
+                from ming_sim.session_write_queue import get_session_write_queue
+                get_session_write_queue(session).complete(ticket)
+            except Exception:
+                pass
 
 
 def minister_chat(session: GameSession, character: Character) -> str:
     """与一位大臣对话。返回 'dismiss' | 'court_break' | 'summon:<name>'。"""
     other = next((n for n in session.content.characters if n != character.name), character.name)
     print(f"\n{character.name}入殿。可持续问话；done/退下 退下，“传{other}来”换人，quit 退朝审阅诏书，exit 退出游戏。")
     print(f"提示：陛下示意采纳后（如“准奏”），大臣会拟旨呈陛下核定。\n")
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)